src/index.html

```diff
@@ -10,7 +10,31 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+
+    <form action="submit"
+    class="search__form"
+    method="submit"
+    data-qa="big">
```

Comment:
```suggestion
    <form 
      action="submit"
      class="search__form"
      method="submit"
      data-qa="big"
    >
```

Reply:
<img width="1057" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/70220667/1fc4e660-21ea-4a27-8d78-6d0a7955d7a1">
fix it everywhere